docs/genealignmentsynteny.md

```diff
@@ -0,0 +1,213 @@
+
+```bash
+cd  synteny/bird/
+
+curl https://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/003/957/565/GCA_003957565.4_bTaeGut1.4.pri/GCA_003957565.4_bTaeGut1.4.pri_genomic.fna.gz -o bTaeGut1_4.fasta.gz
+
+gunzip bTaeGut1_4.fasta.gz
+```
+
+This leaves us with a file called `bTaeGut1_4.fasta` the genomic assembly of bTaeGut1_4 (the Darwin Tree of Life ID for this species) also known as `taeniopygia guttata`, the Australian Zebra Finch.
+
+Now lets move into the raw_data folder and download some data, this may take some time.
+
+```bash
+cd  ../../gene_alignment_prep/raw_data/
+
+curl https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/016/699/485/GCF_016699485.2_bGalGal1.mat.broiler.GRCg7b/GCF_016699485.2_bGalGal1.mat.broiler.GRCg7b_cds_from_genomic.fna.gz -o GallusGallus-GRCg7b.cds.fasta.gz
+
+curl https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/016/699/485/GCF_016699485.2_bGalGal1.mat.broiler.GRCg7b/GCF_016699485.2_bGalGal1.mat.broiler.GRCg7b_genomic.fna.gz -o GallusGallus-GRCg7b.cdna.fasta.gz
+
+curl https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/016/699/485/GCF_016699485.2_bGalGal1.mat.broiler.GRCg7b/GCF_016699485.2_bGalGal1.mat.broiler.GRCg7b_protein.faa.gz -o GallusGallus-GRCg7b.pep.fasta.gz
+
+curl https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/016/699/485/GCF_016699485.2_bGalGal1.mat.broiler.GRCg7b/GCF_016699485.2_bGalGal1.mat.broiler.GRCg7b_rna.fna.gz -o GallusGallus-GRCg7b.rna.fasta.gz
+
+```
+
+Now that's all downloaded we need to prep it. At this point it is all still gzipped (the .gz on the end denotes that the file is compressed) in this format we can't use it. So lets use some bash magic.
+
+This is a for loop written in bash, this will look through the current folder we are in for files ending with .fasta.gz and then gunzip them. This unzips the file, uncompresses it so it is usable, and then runs our python script, `GA_data_prep.py`.
+
+```bash
+for i in *.fasta.gz; do
+gunzip $i;
+python3 GA_data_prep.py ${i/.gz} ncbi 10;
+done
+```
+
+This python script command here, in english, means. Take the file, uncompressed, that I downloaded from ncbi and cut it into pieces. A fasta file looks something like this, with headers (lines starting with `>`) and sequence (The usual ATGC's):
+
+```markdown
+> SCAFFOLD_1_DATA_ON_METHODS
+> ATGCGCATGCATGCATCGACTTCGAGCATCGTAG
+> SCAFFOLD_2_DATA_ON_METHODS
+> ACCAGTGCTAGCTAGCTACGTGTGGGTTTGCCCCGTTT
+```
+
+The headers here will be trimmed, to only essential data that you need in order to fine the sequence in your database of choice.
+
+Fasta file may be made up of anywhere between 10's to many thousands of these. So in the case of our cdna and pep files they need to be cut up to let TreeVal have a chance in reading them all in a small time frame.
+
+cds and rna files will be cut up into 1000 header, sequence pairs per file.
+pep and cdna will be cut up by a number you give or by 100.
+
+This is because the size of pep and cdna files are so much larger.
```

Comment:
```suggestion
Fasta file may be made up of anywhere between 10's to many thousands of these. So in the case of our `cdna` and `pep` files they need to be cut up to let TreeVal have a chance in reading them all in a small time frame.

`cds` and `rna` files will be cut up into 1,000 header-sequence pairs per file. The number given on the command-line is ignored.
`pep` and `cdna` will be cut up by a number you give or by 100.

This is because the size of `pep` and `cdna` files are so much larger.
```